tools/GraphClust/AlignCluster/align_cluster.xml

```diff
@@ -0,0 +1,230 @@
+                  $advanced_opts.param_type.tau
+                  $advanced_opts.param_type.struct_weight
+                  $advanced_opts.param_type.indel_opening
+                  $advanced_opts.param_type.indel
+                  $advanced_opts.param_type.alifold_consensus_dp
+            #end if
+        #end if
+      "
+      && 
+      R-scape --outdir ./RESULTS/ result.aln.sth &&
+      RNAz --locarnate '$alignment_clustal' > alignment.rnaz.out &&
+      #if  str($genomic_cons_opts.genomic_cons_opts_selector) != "no":
+          clustal_to_alma.py '$alignment_clustal' $cluster_all &&
+          EvoFoldV2.sh -o alignment.evofold.out '${alignment_clustal}.ama'
+          #if  str($genomic_cons_opts.genomic_cons_opts_selector) == "hg38_100way":
+             '$__tool_directory__/hg38.100way.nh'
+          #elif str($genomic_cons_opts.genomic_cons_opts_selector) == "hg38_20way":
+             '$__tool_directory__/hg38.20way.nh'
+          #else 
+         '$alignment_clustal'.ama $genomic_cons_opts.phylo_tree
+      #end if
+      &&
+    #else
+      touch alignment.evofold.out &&
+    #end if
+    
+    extract_conservation_metrics.py 
+    '$alignment_clustal' alignment.rnaz.out RESULTS/result.aln.sum alignment.evofold.out  
+    $cluster_all 
+    #if  str($bed_opts.bed_opts_selector) == "yes":
+        $bed_opts.transcript_loci_fasta 
+    '$bed_opts.transcript_loci_bed' 
+    '$bed_opts.genome_version' 
+  #else  
+    '' '' ''
+  #end if
+  conservation_metrics.tsv
+   
+]]>
+  </command>
+  <inputs>
+      <param type="data" name="cluster_all_fa" label="cluster-sequences-sorted" format="fasta"
+        help="Cluster sequences from the collect-result step"/>
+      <param type="data" name="cluster_all" label="cmsearch-results" format="txt"
+        help="Tabular cmsearch results of the cluster-collection step (CLUSTERS-cmsearch)"/>
+      <param name="results_top_num" type="integer" value="5" size="5" label="results_top_num" 
+        help="Top number of cmsearch hit sequences are used for structural alignment and metrics"/>
+
+      <conditional name="genomic_cons_opts">
+          <param type="select" name="genomic_cons_opts_selector" label="Phylo tree for evofold" 
+              help="For MAF genomic alignments as input data, corresponding phylo tree is required to perform Evofold2 structural conservation predcition. For other genomic alignemnts, please upload the tree file.">
+              <option value="no">No</option>
+              <option value="hg38_100way">hg38-100way</option>
+              <option value="hg38_20way">hg38-20way</option>
+              <option value="user_phylo_tree">from history</option>
+          </param>
+          <when value="no" />
+          <when value="hg38_100way" />
+          <when value="hg38_20way" />                
+          <when value="user_phylo_tree">
+              <param format="txt" name="phylo_tree" type="data" label="Newick phylo-tree" />
```

Comment:
we Galaxy knows about a newick format imho

Reply:
Can you check please that all your `formats` are correct ... so maybe tabular instead of txt?

Reply:
uploaded nw files are not detected as nw but as txt ;)

Reply:
https://github.com/galaxyproject/galaxy/blob/dev/config/datatypes_conf.xml.sample#L347 or the nhx filetype. 

There is actually a sniffer: https://github.com/galaxyproject/galaxy/blob/dev/config/datatypes_conf.xml.sample#L805 if you have this fileformt ... but even if this does not work. It should be the user that sets then the datatype.